pkg/divert/nginx/services.go

```diff
@@ -40,11 +40,6 @@ func (d *Driver) divertService(ctx context.Context, name string) error {
 			if !k8sErrors.IsAlreadyExists(err) {
 				return err
 			}
-			// the service was created, refresh the cache
-			newS, err = d.client.CoreV1().Services(d.namespace).Get(ctx, newS.Name, metav1.GetOptions{})
-			if err != nil {
-				return nil
-			}
```

Comment:
Why we need this? Don't we need to refresh the cache (base on the inline comment)

Reply:
👍  perfect, thanks for the explanation!